api/app/Console/Commands/SubmittedAtApplicationDates.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\PoolCandidate;
+use Illuminate\Console\Command;
+use Carbon\Carbon;
+use Database\Helpers\ApiEnums;
+
+class SubmittedAtApplicationDates extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'update-submitted-at';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'For pool candidates that should have submitted dates, submit them';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $dateNow = Carbon::now();
+        $candidatesToSubmit = PoolCandidate::where('pool_candidate_status', ApiEnums::CANDIDATE_STATUS_NEW_APPLICATION)
```

Comment:
But wouldn't this condition make all draft application also to be submitted as well ? 🤔 
I would think this command should be scoped to only those manually added applications.
I'm unsure how though. Probably adding the pool name?

Reply:
should not
created applications have the draft status

https://github.com/GCTC-NTGC/gc-digital-talent/blob/f3cb8731f011704811d5df3c24cda6a935792738/api/app/GraphQL/Mutations/CreateApplication.php#L45-L48

Reply:
ooooo ... I think we thought `CANDIDATE_STATUS_NEW_APPLICATION` is draft status for that production issue. 
![image (15)](https://user-images.githubusercontent.com/89482910/227286832-8449f742-de23-45ec-ab6a-b3c685088f59.png)
Hence we added notDraft scope ..thats why these candidates are not showing up for admins any more 😅

Reply:
I think we should be fine after this. Just need to make sure both draft & new application status are working fine with this change.